### ISSUE TITLE
[Accessibility] Keyboard control using 'Enter' or 'space' in strategy menu does not work

### ISSUE THREAD

**Issue by tiagofilipenunes** [bancorprotocol/carbon-app#1318]
Flow:
1. Go to my strategies
2. Using the keyboard, open the menu and navigate to an option
3. Try to open using enter or space

Current: Nothing happens
Expected: Open the selected option

Tested on Firefox 127.0.1 and Chrome 126.0.6478.127 on MacOS 14.5
